src/Service.GraphQLBuilder/Queries/InputTypeBuilder.cs

```diff
@@ -97,7 +97,7 @@ private static List<InputValueDefinitionNode> GenerateOrderByInputFieldsForBuilt
                     defaultValue: null,
                     new List<DirectiveNode>()));
 
-            inputTypes.Add(
+            inputTypes.TryAdd(
```

Comment:
So if you see this as a sample input: 
![image](https://github.com/Azure/data-api-builder/assets/124841904/9cb3df73-60ee-4b1e-bd54-07c692f4b3da)
I had added the check for type as failsafe for possible duplicate entityNames - you could have two customersObjectsFilterInput generated. but since we validate this in runtimeConfig, I think i can remove this and move it back to the Add.